gr-fec/lib/conv_bit_corr_bb_impl.cc

```diff
@@ -127,7 +127,7 @@ int conv_bit_corr_bb_impl::general_work(int noutput_items,
     uint8_t* score_in = (uint8_t*)input_items[0];
 
     // counting on  1:1 forecast + history to provide enough ninput_items... may need to
-    // insert check printf("%d, %d, %d\n", ninput_items[0], noutput_items, d_counter);
```

Comment:
not sure if the changes to remove newline here help

Reply:
they don't, but they unify things, in case someone wants to convert this to a logging point later on.